ComputerVision/CIFAR/utils/fusion_utils.py

```diff
@@ -0,0 +1,106 @@
+import copy
```

Comment:
we should remove this entire file - this was for internal testing only - we never ended up using it.